README.md

```diff
@@ -3,3 +3,32 @@ Scripts for indexing data into ODC instances
 
 # GDAL3
 Please note - we are aware that using these scripts with GDAL 3.0+ can cause Latitudes and Longitudes to become transposed. When using these scripts, please use a python environment with an earlier version of GDAL.
+
+
+# odc-product-delete (In beta)
+SQL scripts to delete ODC products and related records in ODC, Explorer and OWS DB. Each script takes product_name as an input parameter.
+
+Usage:
+```
+psql -v product_name=<product-to-delete> -f <scriptname.sql> -h <database-hostname> <dbname>
+```
+
+- `delete_odc_product.sql` 
+    - This script will delete a product and all datasets from an ODC DB. 
+    - It deletes records from tables `agdc.dataset_source`, `agdc.dataset_location`, `agdc.dataset` and `agdc.dataset_type`.
+    - It also deletes indexes and view related to the ODC product.
```

Comment:
Hey @jeremyh are indexes and views consistent across metadata types?

Reply:
Yep. They're built based on the metadata type, so products with the same md-type will have the same index+view implementations. But they're created separately for each per-product, so deleting them is still correct. If that's what you're asking.

Reply:
I was just a little uncertain if some products will have views/indexes that are named differently.